src/server.js

```diff
@@ -62,6 +64,23 @@ app.get("/movie/:movieId", async (req, res) => {
     }
 })
 
+app.post("/api/movies/:id/reviews", async (req, res) => {
+    const id = req.params.id;
+    const body = req.body;
+    const movie = await api.getMovie(id);
+    const review = {
+        movie: movie,
```

Comment:
This should not be required, I believe you can just post the ID (so you wouldn't have to retrieve the entire movie object first).